lib/shelley/test/integration/Main.hs

```diff
@@ -165,9 +165,7 @@ testPoolConfigs =
     [ -- This pool should never retire:
       PoolConfig {retirementEpoch = Nothing}
       -- This pool should retire almost immediately:
-    , PoolConfig {retirementEpoch = Just 3}
-      -- This pool should retire, but not within the duration of a test run:
-    , PoolConfig {retirementEpoch = Just 100_000}
+    , PoolConfig {retirementEpoch = Just 1_000}
```

Comment:
Arf! I think there has been a wrong rebase / conflict resolution here! Hence the integration failure.

Reply:
Oops